Pose3D offset   # /**< The estimated offset (including odometry) from last torso matrix to this one. (relative to the torso) */
bool isValid
